File: data/gpt-4o/readable_format_HPEC/cpp/file228.cpp
void print_array(int maxgrid, DATA_TYPE path[][maxgrid])
{
 for(int i=0; i<maxgrid; i++)
 {
  for(int j=0; j<maxgrid; j++)
  {
    printf(DATA_PRINTF_MODIFIER, path[i][j]);
    if(((i*maxgrid)+j)%20 == 19)
    printf("
");
  }
 }
 printf("
");
}

File: data/StarCoder2/readable_format_HPEC/fortran/file228.f90
subroutine print_array(maxgrid, path)
implicit none

integer :: i, j, maxgrid
DATA_TYPE, dimension (maxgrid, maxgrid) :: path
do i = 1, maxgrid
do j = 1, maxgrid
write(0, DATA_PRINTF_MODIFIER) path(j, i)
if (mod(((i - 1) * maxgrid) + j - 1, 20) == 0) then
write(0, *)
end if
end do
end do
write(0, *)
end subroutine
